packages/flutter/lib/src/material/drawer.dart

```diff
@@ -217,7 +234,20 @@ class DrawerControllerState extends State<DrawerController> with SingleTickerPro
   }
 
   void _move(DragUpdateDetails details) {
-    final double delta = details.primaryDelta / _width;
+    if (_controller.isDismissed) {
+      final ModalRoute<dynamic> parentRoute = ModalRoute.of(context);
+      final bool canPop = parentRoute?.canPop ?? false;
+      if (canPop) {
+        Navigator.of(context).pop();
+        return;
+      }
```

Comment:
I don't think this interaction design is of sufficient quality to land.  We can land the patch without an interaction design for the two-drawer case or we can figure out an interaction design for the two-drawer case in this patch, but I'd prefer not to land a patch with a half-designed interaction.

Reply:
Okay. 
I have not been able to get the behavior working via `addStatusListener` on the `_controller`.
Hence, I'm removing this logic from `_move` for now.